notes/2020/2020-07-02.md

```diff
@@ -0,0 +1,86 @@
+# 2020-July-02 ESLint TSC Meeting Notes
+
+## Transcript
+
+[`2020-07-02-transcript.md`](2020-07-02-transcript.md)
+
+## Attending
+
+* Kai Cataldo (@kaicataldo) - TSC
+* Brandon Mills (@btmills) - TSC
+* Nicholas C. Zakas (@nzakas) -TSC
+* Toru Nagashima (@mysticatea) - TSC
```

Comment:
Should we remove this line since @mysticatea was unable to make it?

Reply:
Indeed, good catch.

```suggestion
```